cmd/soroban-rpc/internal/methods/get_ledger_entry.go

```diff
@@ -0,0 +1,83 @@
+package methods
+
+import (
+	"context"
+
+	"github.com/creachadair/jrpc2"
+	"github.com/creachadair/jrpc2/code"
+	"github.com/creachadair/jrpc2/handler"
+
+	"github.com/stellar/go/clients/stellarcore"
+	proto "github.com/stellar/go/protocols/stellarcore"
+	"github.com/stellar/go/support/log"
+	"github.com/stellar/go/xdr"
+)
+
+type GetLedgerEntryRequest struct {
+	Key string `json:"key"`
+}
+
+type GetLedgerEntryResponse struct {
+	XDR                string `json:"xdr"`
+	LastModifiedLedger int64  `json:"lastModifiedLedgerSeq,string"`
+	LatestLedger       int64  `json:"latestLedger,string"`
+}
+
+// NewGetLedgerEntryHandler returns a json rpc handler to retrieve a contract data ledger entry from stellar cre
```

Comment:
nit, comment can simplify to something like 'NewGetLedgerEntryHandler returns a json rpc handler to retrieve the specified ledger entry from stellar core'